calyx-ir/src/control.rs

```diff
@@ -500,6 +507,62 @@ impl StaticControl {
         StaticControl::Empty(Empty::default())
     }
 
+    /// Convience constructor for static enable.
+    pub fn enable(group: RRC<StaticGroup>) -> Self {
+        StaticControl::Enable(StaticEnable {
+            group,
+            attributes: Attributes::default(),
+        })
+    }
+
+    /// Convience constructor for static enable.
+    pub fn seq(stmts: Vec<StaticControl>, latency: u64) -> Self {
+        StaticControl::Seq(StaticSeq {
+            stmts,
+            attributes: Attributes::default(),
+            latency,
+        })
+    }
+
+    /// Convience constructor for static enable.
+    pub fn par(stmts: Vec<StaticControl>, latency: u64) -> Self {
+        StaticControl::Par(StaticPar {
+            stmts,
+            attributes: Attributes::default(),
+            latency,
+        })
+    }
+
+    /// Convience constructor for static if
+    pub fn if_(
```

Comment:
nit: I don't personally love the trailing underscore, maybe something like
`static_if` or `if_static`